src/test/resources/testdata/CAT Subject Access Request API data.json

```diff
@@ -81,8 +81,8 @@
                     "likelyToAbscondText":"Details"
                 },
                 "riskOfHarm": {
-                    "seriousHarm": "Yes",
-                    "harmManaged":"Yes",
+                    "seriousHarm": "Yes"
```

Comment:
would need the trailing comma to be valid json

Reply:
This is not used, this is an exact copy of what came from the ticket, but I want to add it in here for historical purposes. The file with REDACted atthe end of the name is fixed json